fix: update g3doc links (#2273)
tensorflow/docs: site/en/hub/migration_tf2.md

> Many tutorials show these APIs in action. See in particular

Reviewer: ```suggestion
Many tutorials show these APIs in action. Here are some examples:
```
